Documentation/Development/HowTo_HelpSystem.md

```diff
@@ -0,0 +1,3 @@
+# How To: Work with Help System in WheelMUD
+
+TODO.
```

Comment:
Yes I recall some old discussions around this.
IE http://muds.gamedev.com/mudforum/index.php?t=msg&goto=339&rid=0&srch=help#msg_339
I'll revive that thread for discussion.